bentoml/server/api_server.py

```diff
@@ -60,6 +61,28 @@
 '''
 
 
+def log_traceback(func):
+    """
+    Log all the exceptions, using it as the decorator
+    """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except Exception:
+            exception_type, exception_instance, exception_traceback = sys.exc_info()
+            formatted_traceback = "".join(traceback.format_tb(exception_traceback))
+            exception_message = (
+                f"{formatted_traceback}\n{exception_type.__name__}: "
+                f"{exception_instance}"
+            )
+            logger.error(exception_message)
+            raise exception_type(exception_message)
```

Comment:
If you just want to re-raise the original exception, you can do

`except Exception as exc`

at line 73, then, here, just do

`raise exc`